resources/hvac.rb

```diff
@@ -1069,11 +1138,19 @@ def self.apply_mshp(model, runner, seer, hspf, shr,
     d = oat_slope
     e = 0
     f = 0
+
+    a = a - 160.0 / 9.0 * (b + d) + 25600.0 / 81.0 * (c + e + f)
```

Comment:
Why did you copy this code instead of using the convert_curve_biquadratic method?

Reply:
Good question.